### PR TITLE
Update Testcontainers version to latest

### PR DESCRIPTION
**Pull request by rnorth**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -20,7 +20,7 @@ configurations {
 
 dependencyManagement {
     imports {
-        mavenBom 'org.testcontainers:testcontainers-bom:1.9.0'
+        mavenBom 'org.testcontainers:testcontainers-bom:1.10.7'
```
Comment (bsideup) [inline build.gradle]: why not 1.11.2? :D

Comment (rnorth) [reply build.gradle]: Why get it right when I can hilariously get it wrong over and over again?! :D

Thank you - fixed :)

File: step-6-adding-redis.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Step 6: Adding Redis
 
-If you check Testcontainers' modules, you'll see that there is no Redis module \(at least not yet ;\)\): [https://github.com/testcontainers/testcontainers-java/tree/1.8.1/modules](https://github.com/testcontainers/testcontainers-java/tree/1.8.1/modules)
+If you check Testcontainers' modules, you'll see that there is no Redis module \(at least not yet ;\)\): [https://github.com/testcontainers/testcontainers-java/tree/1.10.7/modules](https://github.com/testcontainers/testcontainers-java/tree/1.10.7/modules)
```
Comment (bsideup) [inline step-6-adding-redis.md]:
```suggestion
If you check Testcontainers' modules, you'll see that there is no Redis module \(at least not yet ;\)\): [https://github.com/testcontainers/testcontainers-java/tree/1.11.2/modules](https://github.com/testcontainers/testcontainers-java/tree/1.11.2/modules)
```

File: step-9-selenium-tests.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ public class SystemTest {
 
     @ClassRule
     public static BrowserWebDriverContainer browser = (BrowserWebDriverContainer) new BrowserWebDriverContainer()
-            .withDesiredCapabilities(DesiredCapabilities.chrome())
+            .withCapabilities(DesiredCapabilities.chrome())
```
Comment (bsideup) [inline step-9-selenium-tests.md]: was it a breaking change btw?

Comment (rnorth) [reply step-9-selenium-tests.md]: No, it was non-breaking (deprecation)

**Pull request merged**